sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/PartitionPublishingProperties.cs

```diff
@@ -58,12 +58,14 @@ internal static PartitionPublishingProperties Empty
         /// </summary>
         ///
         /// <value>
-        ///   The sequence number will be in the range of <c>0</c> - <see cref="int.MaxValue"/> (inclusive) and will
-        ///   increase as events are published.  When more than <see cref="int.MaxValue" /> events have been published,
-        ///   the sequence number will roll over to <c>0</c>.
+        ///   <para>The sequence number will be in the range of <c>-1</c> - <see cref="int.MaxValue"/> (inclusive) and will
```

Comment:
How is `-1` different from null?

Reply:
The service explicitly returns -1 when publishing is initialized and no event was previously sent for the configuration.   We do not have a way to query the service for it's state when customers request the properties, only after a publishing operation.   The intent here is to match the initial state of the service with the hope that we'll be able to query and remove the hardcoding before this goes GA.